.changeset/few-nails-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': patch
+---
+
+Add docs section for Kubernetes plugin Auth.
```

Comment:
Looks like you [don't need a changeset](https://github.com/backstage/backstage/pull/13409#issuecomment-1230721848) when you're only changing docs!

Reply:
Removed, thanks.